src/services/products.js

```diff
@@ -15,7 +15,9 @@ export const loadProducts = async (dispatch) => {
         payload: data?.products || [],
       });
     }
+
   } catch (e) {
+    console.log(e);
```

Comment:
console.error instead of console.log

Reply:
Removed the log - [e4f6c13](https://github.com/SukanyaSen530/WeebO-frontend/pull/11/commits/e4f6c1359a136c3468a711cafd7c72212e8396e0)